pkg/ebpf/tracee.go

```diff
@@ -1088,3 +1170,17 @@ func (t *Tracee) invokeInitEvents() {
 		}
 	}
 }
+
+const IoctlSyscallHook int = 65 // randomly picked number for ioctl cmd
+
+func (t *Tracee) invokeIoctlTriggeredEvents() {
+	// invoke DetectHookedSyscallsEvent
+	if t.eventsToTrace[PrintSyscallTableEventID] || t.eventsToTrace[DetectHookedSyscallsEventID] {
+		ptmx, err := os.OpenFile(t.config.Capture.OutputPath, os.O_RDONLY, 444)
+		if err != nil {
+			return
+		}
+		syscall.Syscall(syscall.SYS_IOCTL, ptmx.Fd(), uintptr(IoctlSyscallHook), 0)
```

Comment:
Shouldn't you check if you succeed?

Reply:
Actually no, because we aren't really "talking" to char device (the IOCTL is triggering `/tmp/tracee.pid`), so every time we will try to send that ioctl we will get error that `/tmp/tracee.pid` isnt  char device (but the ioctl event will be triggered)

Reply:
Btw 
This is why in the previous comment, I didn't return error.